attest/verifier/src/avr.rs

```diff
@@ -14,7 +14,7 @@ use mc_attest_core::{IasNonce, VerificationReportData};
 use serde::{Deserialize, Serialize};
 
 /// An enumeration of AVR data verifiers
-#[derive(Clone, Debug, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize)]
+#[derive(Clone, Debug, Eq, Hash, PartialEq)]
```

Comment:
`IsvSvn` doesn't implement `Deserialize, Ord, PartialOrd, Serialize`.  Removing these seemed to have no effect, so I went that route for now. 

I'm kind of curious why these traits were here:
- `Ord` and `PartialOrd` don't seem to make much sense to compare verifier structs to each other, but I think they're required if one is using `BtreeMap`. 
- `Serialize` and `Deserialize` seems interesting, I'm trying to figure out when you'd want to receive a verifier over the wire versus build your own. i.e. can you trust the one over the wire is correct?